### PR TITLE
Auth and dashboard

### PR DESCRIPTION
**Pull request by Midnite**
- NextAuth uses env vars, need an `.env` file

Co-authored-by: Anubha Singh <anubha.singh@red-badger.com>

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -1,15 +1,41 @@
...
+        </Provider>
       </CookiesProvider>
     </ChakraProvider>
   );
 }
 
+function Auth({ children }) {
+  const [session, loading] = useSession();
+  const isUser = !!session?.user;
+  React.useEffect(() => {
```
Comment (siame) [inline pages/_app.js:29]: Any reason to not import `useEffect` above and have this scoped as `React.useEffect`?

File: pages/index.js [hunk 2]
```diff
@@ -15,6 +18,23 @@ export default function Home() {
...
+        {session && (
+          <>
+            <h2>Signed in as {session.user.email}</h2> <br />
+            <button onClick={() => signOut()}>Sign out</button>
+          </>
+        )}
+        <main className={styles.main}>
+          <h1 className={styles.title}>Welcome to JAM</h1>
+          <a href="/dashboard">Click here for the Admin Dashboard</a>
+        </main>
```
Comment (siame) [inline pages/index.js:36]: No more `styles.main` after the changes here: https://github.com/redbadger/jams/pull/48 😢 . Also it looks like this block is repeated with the one on lines 15-19, maybe merge these together using the Chakra components?

File: pages/_app.js (hunk 1 above)
Comment (Midnite) [reply pages/_app.js:29]: no, will change it :)

File: pages/index.js (hunk 2 above)
Comment (Midnite) [reply pages/index.js:36]: this looks kinda like a merge slip up on my side, I got rid of all the styles 🤔

Comment (Midnite) [reply pages/index.js]: yeah that was it, forgot to commit the cleaned up file after rebase

**Review by siame — APPROVED** [review]

**Pull request merged**